metrics/sender/sender.go

```diff
@@ -0,0 +1,25 @@
+package sender
+
+import (
+	"fmt"
+
+	"github.com/openfresh/plasma/config"
+)
+
+type MetricsSender interface {
```

Comment:
__[golint]__
* [golint] exported type MetricsSender should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->